### PR TITLE
Nightly test: beta and canary fail b/c semver comparison

### PR DESCRIPTION
**Pull request by snewcomer**
ref https://www.npmjs.com/package/semver

Screen Shot 2021-08-07 at 9 09 28 AM

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -334,10 +336,3 @@ jobs:
           CI: true
         run: yarn test-external:${{ matrix.partner }}
         continue-on-error: ${{ matrix['continue-on-error'] == true }}
-      - name: upload built assets
-        if: failure() || success()
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.partner }}-dist
-          path: ${{ github.workspace }}/__external-test-cache/${{ matrix.partner }}/dist/assets
-        continue-on-error: ${{ matrix['continue-on-error'] == true }}
```
Comment (snewcomer) [inline .github/workflows/main.yml:343]: We probably don't need this.  We can just run this locally

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -319,9 +319,11 @@ jobs:
             continue-on-error: true
           - partner: model-fragments
             continue-on-error: true
+          - partner: ember-observer
+            continue-on-error: true
```
Comment (snewcomer) [inline .github/workflows/main.yml:323]: This is currently failing.  Ran locally to confirm.  I've seen this error before but not sure exactly what is going on now.

Screen Shot 2021-08-08 at 2 13 43 PM

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -228,11 +228,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scenario: [ember-release, ember-beta, ember-canary]
+        scenario: [ember-release, ember-beta]
```
Comment (snewcomer) [inline .github/workflows/main.yml:231]: canary is timing out.  If canary is failing now, beta will soon as well.  Tests just aren't starting up.

Comment (snewcomer) [reply .github/workflows/main.yml:231]: I think we talked about potentially excluding this.  But perhaps I'm misremembering what that was.

Comment (igorT) [reply .github/workflows/main.yml:231]: Yeah that makes sense, we probably want to not have `beta` block CI either.

File: .github/workflows/main.yml (hunk 1 above)
Comment (igorT) [reply .github/workflows/main.yml:343]: I think the idea was it makes debugging faster if I understood correctly, no?

Comment (snewcomer) [reply .github/workflows/main.yml:343]: Likely.  We do have those scripts `test-external:*` to run the specific failures and debug just as easily as well.

**Pull request merged**